releng/org.eclipse.xtext.contributor/Xtext.setup

```diff
@@ -80,32 +80,33 @@
   </setupTask>
   <setupTask
       xsi:type="setup:VariableTask"
-      name="version.mwe2"
-      value="2.9.0"/>
```

Comment:
is this no longer needed?

Reply:
No, since we use the composite sites. This includes MWE and Xpand.